src/alert/lsst.rs

```diff
@@ -193,10 +193,10 @@ impl TryFrom<DiaSource> for Candidate {
             .ok_or(AlertError::MissingFluxAperture)?
             * 1e-9;
 
-        let (magpsf, sigmapsf) = flux2mag(psf_flux.abs(), psf_flux_err);
-        let diffmaglim = fluxerr2diffmaglim(psf_flux_err);
+        let (magpsf, sigmapsf) = flux2mag(psf_flux.abs(), psf_flux_err, 8.9);
```

Comment:
If the LSST zero point is always 8.9, I would make it a `const` this module.

Reply:
done